packages/design-toolkit/src/components/search-field/styles.module.css

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+@reference '../../index.css';
+
+@layer components.l1 {
+  .field {
+    @apply fg-primary-bold flex items-center gap-s rounded-round p-s outline;
+
+    @variant hover {
+      @apply outline-interactive-hover;
+    }
+
+    @variant focus-visible-within {
+      @apply outline-accent-primary-bold;
+    }
+
+    @variant pressed {
+      @apply outline-interactive-pressed;
+    }
+
+    @variant disabled {
+      @apply fg-disabled cursor-not-allowed outline-interactive-disabled;
+    }
+
+    &.filled {
+      @apply bg-surface-raised outline-static;
+    }
+
+    &.outline {
+      @apply outline-interactive;
+    }
+  }
+
+  .input {
+    @apply block w-full font-display text-body-s outline-none;
+
+    /* TODO: hide-cancel utility class is missing */
+    &::-webkit-search-cancel-button {
+      @apply hidden;
+    }
+
+    @variant placeholder-shown {
+      @apply fg-primary-muted;
+    }
+
+    @variant disabled {
+      @apply fg-disabled cursor-not-allowed;
+    }
+  }
+
+  .clear {
+    @apply fg-info-bold cursor-pointer;
+
+    @variant hover {
+      @apply fg-info-hover;
+    }
+
+    @variant focus-visible {
+      @apply fg-info-hover;
+    }
+
+    @variant group-empty/search-field {
+      @apply invisible;
+    }
+
+    @variant group-disabled/search-field {
+      @apply invisible;
+    }
+  }
+
+  .loading {
+    /* TODO: Not animation, TW animation CSS vars seem to be missing */
```

Comment:
i was going to say we _could_ inline it if `@apply` is acting funny
```
      animation: var(--animate-spin, spin 1s linear infinite);
```

but oddly enough, it's working for me locally

<img width="1761" height="788" alt="Screenshot 2025-11-17 at 11 35 53 AM" src="https://github.com/user-attachments/assets/f0d0c11a-ae29-4032-81c4-41f853a76813" />